src/Sulu/Bundle/ContentBundle/Controller/NodeController.php

```diff
@@ -360,9 +360,6 @@ public function putAction(Request $request, $uuid)
         $webspace = $this->getWebspace($request);
         $template = $this->getRequestParameter($request, 'template', true);
 
-        $isShadow = $this->getRequestParameter($request, 'shadowOn', false);
-        $shadowBaseLanguage = $this->getRequestParameter($request, 'shadowBaseLanguage', null);
```

Comment:
But this is in a different method, this variables are also still used in this one. (see line 378 & 379)